pages/edit.js

```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components";
+import Link from "next/link";
+import UpdateFlashcard from "@/components/UpdateFlashcard";
+import { useRouter } from "next/router";
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+export default function Edit({ handleUpdateFlashcard, flashcards }) {
+  const router = useRouter();
+  const { id } = router.query;
```

Comment:
use the name of the dynamic route variable here :
```suggestion
  const { cardId } = router.query;
```

Reply:
Done